QubitServer/src/main/java/org/labrad/qubits/QubitContext.java

```diff
@@ -21,7 +21,7 @@
 import org.labrad.qubits.channeldata.*;
 import org.labrad.qubits.channels.*;
 import org.labrad.qubits.config.MicrowaveSourceConfig;
-import org.labrad.qubits.config.MicrowaveSourceOffConfig;
+import org.labrad.qubits.config.MicrowaveSourceOffConfig$;
```

Comment:
That's right; this and the `MODULE$` business below will both go away when the conversion is done.